apps/client/src/app/career-qna/page.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+
+import styled from '@emotion/styled';
+import { Flex, Stack, Text } from '@sickgyun/ui';
+import { useState } from 'react';
+import Footer from '@/components/common/Footer';
+import Header from '@/components/common/Header';
+import QnaBox from '@/components/qna-posting/QnaBox';
+import QnaListBox from '@/components/qna-posting/QnaListBox';
+import QnaPostingList from '@/components/qna-posting/QnaPostList';
+import { QNA_LIST } from '@/constants/qna';
+
+const CareerQna = () => {
+  const [currentQnaPage, setCurrentQnaPage] = useState(0);
+
+  const handlePrevCareerQna = () => {
+    if (currentQnaPage > 0) {
+      setCurrentQnaPage((prev) => prev - 1);
+    }
+  };
+
+  const handleNextCarrerQna = () => {
+    if (currentQnaPage < QNA_LIST.length - 3) {
+      setCurrentQnaPage((prev) => prev + 1);
+    }
+  };
+
+  return (
+    <>
+      <Header />
+      <StyledCareerQnaLayout>
+        <StyledCareerQna>
+          <Stack direction="vertical" spacing={15} style={{ marginBottom: '60px' }}>
+            <Flex justify="space-between">
+              <Text fontType="h3">🔥 인기글</Text>
+              <Stack direction="horizontal" spacing={10}>
+                <Stack onClick={handlePrevCareerQna}>
+                  <StyledArrowImage src="/assets/arrow_left.png" />
```

Comment:
```suggestion
                  <StyledArrowImage src="/assets/arrow_left.png" />
```

png말고 icon이니깐 [다음](https://github.com/sickgyun/sickgyun-client/blob/main/packages/ui/src/Icons/ArrowBack.tsx)과 같은 svg 컴포넌트로 사용하면 더 유연한 아이콘이 되어 많은 곳에서 사용 할 수 있을것가타용

Reply:
넵